data/templated/example.xml

```diff
@@ -61,4 +61,16 @@
          <value>11</value>
       </values>
    </template>
+   <template name="GetItemRandom">
+      <query><![CDATA[SELECT COUNT(*) FROM item WHERE i_im_id = ?]]></query>
+      <types>
+         <type>RANDOM</type>
+      </types>
+      <values>
+         <value>binomial,0,9999</value>
```

Comment:
Would be good to have a test case for each of your supported types.
That would also help document a little better how this should work.
Another nice documentation item could be to expand on the `README.md` in the templated benchmarks source directory.

Reply:
Will do as soon as I publish the PR :+1:

Reply:
I think the documentation should be updated as part of the PR to be consistent.